stockflux-watchlist/README.md

```diff
@@ -1 +1,43 @@
-# StockFlux Watchlist
+## StockFlux Watchlist
+
+StockFlux Watchlist is a part of StockFlux app suite intended to work with StockFlux Launcher, Chart, and News apps.
+Watchlist is an app containing the list of symbols the user is watching. These symbols are displayed as cards containing key symbol information and a minichart of the symbol's price fluctuations in the last 30 days.
+
+### Running locally
+
+In order to launch the StockFlux Watchlist app locally, StockFlux Core and StockFlux Components have to be built first.
+
+```
+cd stockflux-core
+npm run build
+cd stockflux-components
+npm run build
+```
+
+Then install StockFlux Watchlist dependencies.
+
+```
+cd stockflux-watchlist
+npm install
+```
+
+To start the StockFlux Watchlist:
+`npm start`
```

Comment:
I guess this starts up the dev-server but doesn't launch the watchlist, we should probably add `npm run launch` into this example snippet too

Reply:
Fixed -> 3893570